test/config/checkov.yaml

```diff
@@ -0,0 +1,20 @@
+---
+check: []
+compact: false
+evaluate-variables: true
+external-checks-dir:
+  - docs
+  - exploration/argocd-external/docs
+framework:
+  - kubernetes
+  - dockerfile
+  - github_actions
+  - gitlab_ci
+  - kustomize
+  - argo_workflows
```

Comment:
I don't think we use argo workflows, do we?

Reply:
you are right.  We don't have the below  argo workflows.  Fixed.

```
apiVersion: argoproj.io/v1alpha1
kind: Workflow
```